bin/plotting/pycbc_page_ifar_vs_stat

```diff
@@ -0,0 +1,168 @@
+                         " decimation to coincs with stat value below the"
+                         " given thresholds. Supply as a comma-separated list"
+                         " of threshold:decimation value pairs surrounded by"
+                         " square brackets (no spaces!). Decimation values must"
+                         " be positive integers."
+                         " Ex. [15:5,10:30,5:30,0:30]."
+                         " Default: no decimation")
+opts = parser.parse_args()
+init_logging(opts.verbose)
+
+sig_dict = significance.digest_significance_options(opts.ifo_combos, opts)
+
+coinc_color_map = {
+    'H':  '#ee0000',  # red
+    'L':  '#4ba6ff',  # blue
+    'V':  '#9b59b6',  # magenta/purple
+    'HL': '#0000DD',
+    'LV': '#FFCC33',
+    'HV': '#990000',
+    'HLV': np.array([20, 120, 20]) / 255.}
+
+coinc_line_map = {
+    'H':  '--',
+    'L':  '--',
+    'V':  '--',
+    'HL': '-',
+    'LV': '-',
+    'HV': '-',
+    'HLV':'-'
+}
+
+exc_zlag_filenames = []
+# Not sure what this step does precisely - select the first matching file ?
+for coinc_key in [f'/{k}-' for k in opts.ifo_combos]:
+    exc_zlag_filenames.append([f for f in opts.trigger_files if coinc_key in f][0])
+logging.info('Got files: ', ', '.join(exc_zlag_filenames))
+
+loudest_keep_vals = opts.loudest_keep_values.strip('[]').split(',')
+threshes = []
+factors = []
+for decstr in loudest_keep_vals:
+    thresh, factor = decstr.split(':')
+    if float(factor) % 1:
+        raise RuntimeError("Non-integer decimation is not supported")
+    if int(factor) < 1:
+        raise RuntimeError("Negative or zero decimation does not make sense")
+    if int(factor) == 1:
+        continue
+    threshes.append(float(thresh))
+    factors.append(int(factor))
+
+# Sort the threshes into descending order
+# - allows decimation factors to be given in any order
+threshorder = np.argsort(threshes)[::-1]
+decimation_stat = np.array(threshes)[threshorder]
+decimation_fact = np.array(factors)[threshorder]
+
+fig = plt.figure(figsize=(9, 6))
+ax = fig.add_subplot(111)
+
+ifar_calc_points = np.linspace(-10, 30, 100)
+
+singles = []
+for exc_zlag_filename in exc_zlag_filenames:
+    logging.info(f"opening file {exc_zlag_filename}")
+    with h5py.File(exc_zlag_filename,'r') as exc_zlag_f:
+        event_ifos = exc_zlag_f.attrs['ifos'].split()
+        coinc_key = ''.join(event_ifos).replace('1', '')
+        stat_exc = exc_zlag_f['background_exc']['stat'][:]
+        stat = exc_zlag_f['background']['stat'][:]
+        dec_fac = exc_zlag_f['background']['decimation_factor'][:]
+        dec_fac_exc = exc_zlag_f['background_exc']['decimation_factor'][:]
+        bg_time = convert_s_to_y(exc_zlag_f.attrs['background_time_exc'])
+        ifar_file = exc_zlag_f['background_exc']['ifar'][:]
+
+    logging.info(f"Min {coinc_key} IFAR: {min(ifar_file)}")
+    logging.info(f"{coinc_key} background time (y): {bg_time}")
+    if len(event_ifos) == 1:
```

Comment:
It may be worth doing `if sig_dict[''.join(event_ifos)]['method'] == 'trigger_fit'`, so this is applicable in case we use trigger fit for coincs